### PR TITLE
[#43906] Single source of truth for GUI backup feature in Cloud and O…

### PR DESCRIPTION
**Pull request by ghost**
…n-premises

[#43906] Single source of truth for GUI backup feature in Cloud and On-premises

https://community.openproject.org/work_packages/43906

### CODE REVIEW

File: docs/system-admin-guide/backup/README.md [hunk 1]
```diff
@@ -7,28 +7,46 @@ keywords: system backup
 ---
 # Backup
 
-Unless disabled via the [configuration](../../installation-and-operations/configuration/#backup-enabled)
-users can make backups of the OpenProject installation from within the administration area.
-They either need to be an administrator or have the global permission to do so.
+## Enterprise cloud
```
Comment (machisuji) [inline docs/system-admin-guide/backup/README.md]: I would suggest putting this section (lines 10 to 14) in the the enterprise cloud guide and link to this page after that. Something like "Please refer to the system admin guide here how to learn how to perform backups" or similar.

File: docs/enterprise-guide/enterprise-cloud-guide/backups/README.md [hunk 2]
```diff
@@ -8,36 +8,11 @@ keywords: backups
 
 # Backups
 
-Your Enterprise cloud data is backed up continuously and retained for 30 days.
-Within those 30 days we can restore your data to any point in time with a precision of 5 minutes, in case you need us to.
+*For a single source of truth, we moved this information*
```
Comment (machisuji) [inline docs/enterprise-guide/enterprise-cloud-guide/backups/README.md:19]: If we put the section here as proposed below we can just drop this line.

**Review by machisuji — CHANGES_REQUESTED** [review]
Hey @adam-op. Thanks for consolidating this. I have a suggestion about the enterprise cloud section. What do you think?

File: docs/enterprise-guide/enterprise-cloud-guide/backups/README.md (hunk 2 above)
Comment (ghost) [reply docs/enterprise-guide/enterprise-cloud-guide/backups/README.md:19]: i think that it is important to link to the Backup Via GUI from the Enterprise Cloud Backup page as before there was the whole process documented and now it is missing there...

Comment (ghost) [reply docs/enterprise-guide/enterprise-cloud-guide/backups/README.md:19]: @machisuji please check

**Comment by ghost** [comment]
@machisuji 
please check

File: docs/enterprise-guide/enterprise-cloud-guide/backups/README.md (hunk 2 above)
Comment (machisuji) [reply docs/enterprise-guide/enterprise-cloud-guide/backups/README.md:19]: You're right, it's important to link there. I still think the sentence "For a single source of truth, we moved this information" can go as it doesn't add anything useful for users. By linking the section in the line below it is obvious that it has been moved for people who were used to the text there and for new users it doesn't matter that it was moved. They just see that it is linked and that's fine.

So I say keep line 21 (Please navigate to...) and drop line 19 (For a single source...)

**Review by machisuji — CHANGES_REQUESTED** [review]
Thanks @adam-op, looks good. I'd still want just a single change, please.

**Comment by ghost** [comment]
@machisuji sorry, but i lost my branch, how to proceed here? can i edit on github? because it is a simple edit, could you edit that please?

**Pull request merged**